src/components/Header.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+import styled from 'styled-components/macro'
+import { ReactComponent as Logo } from '../images/logo.svg'
+
+export default function Header({ headline }) {
+  return (
+    <HeaderWrapper>
+      <LogoStyled />
+      <HeaderTitle>{headline}</HeaderTitle>
+    </HeaderWrapper>
+  )
+}
+
+const HeaderWrapper = styled.div`
```

Comment:
How about using the `<header>` element instead of `<div>`? It would be semantically correct.
And then you could just name it `HeaderStyled`

Reply:
If you name it `HeaderStyled`, I would expect to have a styled version of `components/Header`. 
Maybe rename this module to `AppHeader.js` to avoid this naming conflict.

Personally, I don't like the appendix `Styled` because it doesn't give semantic value. 
It helps if you think about CSS class names.

Example:
```jsx
export default function Header({ headline }) {
  return (
    <header className="header-styled">
      <img className="logo-styled" />
      <h1 className="header-title">{headline}</h1>
    </header>
  )
}
```

What do you think about these class names?

```jsx
export default function Header({ headline }) {
  return (
    <header className="header">
      <img className="logo" />
      <h1 className="title">{headline}</h1>
    </header>
  )
}
```

And in React:

```jsx
export default function AppHeader({ headline }) {
  return (
    <Header>
      <Logo />
      <Title>{headline}</Title>
    </Header>
  )
}
```

Reply:
That is a good point. Thank you for pointing that out!

Reply:
Renaming Logo from `<LogoStyled/>` to `<Logo/>` doesn't works. 
I had it bevor. I get the error message "Identifier 'Logo' has already been declared"
`import { ReactComponent as Logo } from '../images/logo.svg'`
What's the right way to use Naming `<Logo/>`?

Reply:
It won't work because you imported it as `Logo`
What you could do is add the svg styles to your header styles:
```
const Header = styled.header {
 some styles

 svg {
 some styles
 }
}
```

Reply:
thx Lene for your answer. I will try it.